### DISCUSSION TITLE
Testing with zustand and mocha

### DISCUSSION THREAD

**Discussion by nezouse** [pmndrs/zustand#1575] [Q&A]
Hello!

I wanted to use zustand in one project we are actively developing, but not sure how can i easily reset state between tests as we are using Mocha.
I'm aware of [this](https://github.com/pmndrs/zustand/blob/main/docs/guides/testing.mdx#resetting-state-between-tests-in-react-dom) example, but still not sure how to achieve it in mocha.

**Comment by dbritto-dev** [comment]
@nezouse si sabes como hacer mock de modules en mocha puedo ayudarte a modificarlo para que sea compatible con mocha
